lib/Service/SettingsService.php

```diff
@@ -57,6 +57,9 @@ class SettingsService {
 	const DEFAULT_SENSITIVITY = '0.4';
 	const MAXIMUM_SENSITIVITY = '0.6';
 
+	/** Model Path used for read and writing models  **/
+	const SYSTEM_MODEL_PATH = 'modelPath';
```

Comment:
const SYSTEM_MODEL_PATH = 'modelPath';
```suggestion
	const SYSTEM_FACE_RECOGNITION_MODEL_PATH = 'FaceRecognitionModelPath';
```

Reply:
Keep in mind that this configuration will go in `config/config.php` and it should be clear.

Reply:
D'OH!, Again.